Tasks/AzureFileCopy/README.md

```diff
@@ -54,6 +54,8 @@ If the VMs have been created without opening the WinRM HTTPS ports then follow t
 
 The parameters of the task are described in details, including examples, to show how to input the parameters. The parameters listed with a * are required parameters for the task:
 
+* **Azure Connection Type**: Specify Azure EndPoint type, for Azure Classic resources use 'Azure' endpoint, for Azure ARM, use 'Azure Resource Manager'. 
+
 * **Azure Subscription**: The name of Azure subscription, where the Azure storage account is located. The storage account is accessed using the stored credentials of the Azure account in the Services tab. 
```

Comment:
Since we have Azure Classic Subscription n Azure RM subscription based on azure connection type selected, So should we add details for such parameters? others are classic storage account/rm storage account; resource group/cloud services; ?